### PR TITLE
Add new issue forms

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Use new issue form to make reporting bugs, requesting features and topics more precise and easy to fit in infos.

# Open Questions

- [ ] Do we need more options, or is bug report, feature and topic request enough?
- [ ] Do we want to add additional info in the issue interface? Like for example the discord link [here](https://github.com/iotaledger/template/issues/new/choose)
- [ ] Which labels should we use?

### CODE REVIEW

**Comment by jlvandenhout** [comment]
I think adding an option to create an issue specifically to content would make sense. So we have bug reports and feature requests for Wiki functionality and documentation issues and topic requests for content? Didn't think of a good name for that yet though.

**Comment by jlvandenhout** [comment]
For labels we then would have: `bug`, `feature`, `topic` and `content` as category labels.

I was never fond of priority labels, but I'm ready to be convinced. Also I don't see the point in having status labels like `won't fix` or `duplicate`, because that would be something we comment as reason for closing the issue anyway, but again I'm open for discussion there.

Labels like `needs information` and `confirmed`/`accepted` I think would be helpful, as those are all status while the issue is still open.

**Comment by Dr-Electron** [comment]
> For labels we then would have: `bug`, `feature`, `topic` and `content` as category labels.
> 
> I was never fond of priority labels, but I'm ready to be convinced. Also I don't see the point in having status labels like `won't fix` or `duplicate`, because that would be something we comment as reason for closing the issue anyway, but again I'm open for discussion there.
> 
> Labels like `needs information` and `confirmed`/`accepted` I think would be helpful, as those are all status while the issue is still open.

Yeah we can work on that. Those are mostly the default ones from Github. I asked more for the issue templates.
But for sure we also need to clean up our labels 😅

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@
     <a href="https://wiki.iota.org"><strong>Explore the Wiki »</strong></a>
```
Comment (lucas-tortora) [inline README.md:23]: Just out of curiosity, why is the readme using html instead of md?

**Review by lucas-tortora — APPROVED** [review]

**Pull request closed without merging**